CMakeLists.txt

```diff
@@ -645,27 +645,27 @@ endif()
 if(UNIX)
   if(APPLE)
     if (ARCH MATCHES "arm64")
-      set(ENERGYPLUS_EXPECTED_HASH e5a071a01c130aff04515641b502b1ed)
+      set(ENERGYPLUS_EXPECTED_HASH 7301caad82ed104f23704e6123345989)
       set(ENERGYPLUS_PLATFORM "Darwin-macOS13-arm64")
     else()
-      set(ENERGYPLUS_EXPECTED_HASH 21c6e2c536737ef837c2f58c241133cc)
+      set(ENERGYPLUS_EXPECTED_HASH 8746aaad0e901f640b644ffc96d4cde9)
       set(ENERGYPLUS_PLATFORM "Darwin-macOS12.1-x86_64")
     endif()
   elseif(LSB_RELEASE_ID_SHORT MATCHES "CentOS")
-    set(ENERGYPLUS_EXPECTED_HASH 53aef5bb832ad7f1564ea568a60b4d12)
+    set(ENERGYPLUS_EXPECTED_HASH TODO_TBD_TODO)
```

Comment:
@anchapin we're missing a centos build for it

Reply:
Nevermind, didn't realize that this was memorial day. I just built it myself (I think I only have sometimes issues building the OS SDK centos package, never E+)